src/agents/workflows/review_logic.py

```
@@ -132,31 +159,28 @@ def run_review(state: Dict[str, Any]) -> Dict[str, Any]:
+            f"Feature summary: {json.dumps(feature_summary, ensure_ascii=False, default=str)}\n"
+            f"Please return JSON with keys feedback (short) and suggestions (list)."
+        )
 
         parsed = None
         text = ""
         try:
-            ai_msg = model.invoke(prompt)
-            text = getattr(ai_msg, "content", "") or str(ai_msg)
+            text = call_model(MODEL_NAME, REVIEW_SYSTEM_PROMPT, user_prompt)
             parsed = _extract_json(text)
         except Exception:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)